.experimenter.json

```diff
@@ -1,26 +1,4 @@
 {
-  "homescreen": {
```

Comment:
nit: lets not check in this file as part of the PR :)

Reply:
let me remove this file from commit

Reply:
done